client/introduction.mdx

```diff
@@ -111,66 +118,71 @@ All Pipecat client SDKs provide:
     });
 
     try {
-      await rtviClient.connect();
+      // you can provide your Daily connection credentials directly as
+      // part of connect or provide a REST endpoint for acquiring them
+      await pcClient.connect({
+        endpoint: "https://your-connect-end-point-here/connect",
+      });
     } catch (e) {
       console.error(e.message);
     }
 
-    // Events
-    rtviClient.on(RTVIEvent.Connected, () => {
+    // Alternative Event Handling Setup
+    pcClient.on(RTVIEvent.Connected, () => {
       console.log("[EVENT] User connected");
     });
-    rtviClient.on(RTVIEvent.Disconnected, () => {
+    pcClient.on(RTVIEvent.Disconnected, () => {
       console.log("[EVENT] User disconnected");
     });
     ```
 
+    ```react react
+    // Example: Using PipecatClient in a React component
+    ```
+
+    </CodeGroup>
+
 </Tab>
-<Tab title="Single-turn Actions">
-  Send individual actions for:
-  - Text-to-text chat
-  - One-off image processing
-  - Batch operations
-
-```javascript
-// Example: Sending a single action
-import { RTVIClient } from "realtime-ai";
-
-const rtviClient = new RTVIClient({
-  params: {
-    baseUrl: "https://your-connect-end-point-here",
-    endpoint: {
-      connect: "/connect",
-      action: "/action",
-    },
-  },
-  callbacks: {
-    onBotText: (text: BotLLMTextData) => {
-      console.log("Text response:", text);
-    },
-  },
-});
-
-try {
-  await rtviClient.action({
-    service: "llm",
-    action: "append_to_messages",
-    arguments: [
-      {
-        name: "messages",
-        value: [
-          {
-            role: "user",
-            content: "tell me a joke",
-          },
-        ],
+<Tab title="Custom Messaging">
+  Send custom messages and handle responses from your bot. This is useful for:
+  - Running server-side functionality
+  - Triggering specific bot actions
+  - Querying the server
+  - Responding to server requests
+
+  <CodeGroup>
+
+  ```javascript javascript
+  import { PipecatClient } from "@pipecat-ai/client-js";
+
+  const pcClient = new PipecatClient({
+    transport: DailyTransport.create(),
+    callbacks: {
+      onBotConnected: () => {
+        pcClient.sendClientRequest('get-language')
+          .then((response) => {
+            console.log("[CALLBACK] Bot using language:", response);
+            if (response !== preferredLanguage) {
+              pcClient.sendClientMessage('set-language', {language: preferredLanguage});
+            }
+          })
+          .catch((error) => {
+            console.error("[CALLBACK] Error getting language:", error);
+          });
+      },
+      onServerMessage: (message) => {
+        console.log("[CALLBACK] Received message from server:", message);
       },
-    ],
+    },
   });
-} catch (e) {
-  console.error(e.message);
-}
-```
+  await pcClient.connect({url: "https://your-daily-room-url", token: "your-daily-token"});
```

Comment:
Like that you found a way to also illustrate the alternative method of connecting 👍